scripts/calculate_ram_hours.py

```diff
@@ -0,0 +1,89 @@
+"""A script to calculate how many RAM-hours a job took.
+
+A RAM-hour shall be defined as using 1GB of random access memory for 1 hour.
+
+This is a useful metric for two reasons:
+  * Some jobs are fast but take a lot of RAM, others might be slow but
+    only take a bit of RAM. A job that takes 10 minutes using 6GB of
+    RAM and a job that takes an hour using only 1GB of RAM cost us the
+    same.
+  * Based on the amount of RAM an instance has and its cost, we can
+    determine the cost of a ram-hour and use that for estimating the
+    costs of jobs.
+"""
+
+import argparse
+import datetime
+
+import pyrefinebio
+import pytz
+
+
+def parse_args():
+    description = """This script calculates how many RAM-hours a sample has consumed.
+
+    A RAM-hour shall be defined as using 1GB of random access memory for 1 hour.
```

Comment:
I think you have to un-indent these lines, because the leading spaces on each line are stored as part of the string, so when the description is printed it will look like this:
```
This script calculates how many RAM-hours a sample has consumed.

    A RAM-hour...

    All downloader...
```

Reply:
It actually strips out all the whitespace so it looks like:
```
$ python3 ./scripts/calculate_ram_hours.py -h
usage: calculate_ram_hours.py [-h] [--start-date START_DATE] accession_code

This script calculates how many RAM-hours a sample has consumed. A RAM-hour
shall be defined as using 1GB of random access memory for 1 hour. All
downloader and processor jobs that were run will be considered, regardless of
whether they were successful or not. The goal is to determine how many
resources were spent to download and process a sample, regardless of success.

positional arguments:
  accession_code        Specify the accession_code of the sample to calculate
                        RAM-hours for.

optional arguments:
  -h, --help            show this help message and exit
  --start-date START_DATE
                        Do not include jobs started previous to start_date in
                        the calculations. The format is expected to be YYYY-
                        MM-DD.
```

Reply:
But I guess I don't need that extra newline there, although I think it makes it a bit easier to read from within the code.

Reply:
I should have just put that in the PR description to start with! Next time!